_blueprints/work_item_types.adoc

```diff
@@ -0,0 +1,78 @@
+= Work Items and Work Item Types (Proposal)
+This is a working document to come up with a system for describing and defining types of work items. 
+
+== Vision
+There is a central work item (micro-)service that serves as a common platform for storage and query of work items. Client services extend the basic work item types to offer business functionality like different ways of planning (scrum, waterfall, etc.), testing, ideation, etc. These service in turn are the basis for our UI.

```

Comment:
You're thinking that there will be a "Scrum Endpoint" that the UI talk to? The "Scrum Endpoint" then talks to Core backbone?